File: 1. 단계별로_풀어보기/10~19/14. 집합과 맵/7. 1269.js
// 입력
const fs = require('fs')
const inputArr = fs.readFileSync(0).toString().trim().split('\n')

// 입력 파싱
const [numA, numB] = inputArr[0].split(' ').map(Number)
const AArr = inputArr[1].split(' ').map(Number)
const BArr = inputArr[2].split(' ').map(Number)

// A 원소의 집합화
const ASet = {}
AArr.forEach(val => (ASet[val] = 1))

// A 와 B 의 교집합의 원소 갯수
const shareNum = BArr.filter(val => ASet[val] === 1).length

// 결과 출력
process.stdout.write(`${numA + numB - 2 * shareNum}`)

File: 1. 단계별로_풀어보기/10~19/14. 집합과 맵/7. 1269.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

N, M = map(int, input().split())

ASet = set(map(int, input().split()))
BSet = set(map(int, input().split()))

ABSet = ASet & BSet

result = len(ASet) + len(BSet) - 2 * len(ABSet)

output(f"{result}")